libdnf/rpm/solv/goal_private.hpp

```diff
@@ -24,13 +24,22 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "solv_map.hpp"
 
 #include "libdnf/rpm/solv_sack.hpp"
+#include "libdnf/utils/utils.hpp"
 
 #include <solv/solver.h>
 
 namespace libdnf::rpm::solv {
 
 class GoalPrivate {
 public:
+    struct UnresolvedGoal : public LogicError {
+        using LogicError::LogicError;
```

Comment:
The `UnresolvedGoal()` constructor is defined. The line `using LogicError::LogicError;` is no longer needed.